src/renderers/shared/fiber/ReactFiberContext.js

```diff
@@ -57,6 +57,18 @@ function getUnmaskedContext(workInProgress : Fiber) : Object {
 }
 exports.getUnmaskedContext = getUnmaskedContext;
 
+function cacheContext(workInProgress : Fiber, unmaskedContext : Object, maskedContext : Object) {
+  if (isContextConsumer(workInProgress)) {
```

Comment:
I assume that `getMaskedContext` returns `emptyObject` instead of `null` for a reason.

> We could but it seems a bit fishy to make one perf optimization to rely on another perf optimization being present in another file.

This all inside of `ReactFiberContext`. What's the other file?